src/backend/aspen/app/views/phylo_trees.py

```diff
@@ -53,16 +53,6 @@ def phylo_trees():
     phylo_runs: Iterable[Tuple[PhyloRun, int]] = (
```

Comment:
```suggestion
    phylo_runs: Iterable[Tuple[PhyloRun, int]] = (
```

This is probably what py-lint is mad about